docs/docs/90-glossary.md

```diff
@@ -30,3 +32,17 @@ With the Lens Developer Kit (LDK), you can create your own lens based on your sp
 
 ## Perspective
 A Perspective displays Subject and Sample data for some set of Subjects and Aspects you care about, using the Lens you select.
+
+## Bots
+A Bot is an entity capable of performing actions. Bots can interact with external services and might exist outside of Refocus on its own. They can provide graphical user interfaces and access to external APIs.
+
+In Refocus, Bots advertise actions they're capable of doing. Actions performed are logged as events. Bots are distributed and can live inside private networks.
+
+You can create your own bots and register them in Refocus.
+
+## Rooms
+A Room is a place where Bots and people can coexist to perform some actions. Rooms can have different types. Each Room Type defines interaction rules between Bots.
+
+In Refocus, Rooms can be created when a Sample is in undesired condition and action is required. Rooms understand the Refocus hierarchy.
+
+*Disclaimer*: When bots are smart enough, people aren't necessary to enter rooms.
```

Comment:
this isn't a "disclaimer" ... we're not denying responsibility for anything here